### DISCUSSION TITLE
Audi smartphone interface is not created

### DISCUSSION THREAD

**Discussion by cebin-KR** [Mr-MIBonk/M.I.B._More-Incredible-Bash#26] [Q&A]
The software version of my car (AUDI A7 2016) is MHI2_KR_AU57x_S0035.
I've successfully installed the M.I.B. Then, I've executed the AIO patch using MHI2_KR_AU57x_S0035_MU0035_PATCH without any errors.
But, the Audi smartphone interface is not showed up. I've rebooted the system several times to get the same results.

Am I missing something? I hope someone can help me. Best regards.

**Comment by CaneTLOTW** [comment]
Have you checked yout USB port, does it support smartphone interface?

**Reply by jessecpro** [reply]
Carplay doesn't work because carplay is not included in MHI2_KR_AU57x_S0035 firmare. What you need is firmware MHI2_US_AU57x_K3341 MU1426. Grab that firmware and patch. Good luck!
